frontend/device/kobo/nickel_conf.lua

```diff
@@ -53,9 +53,7 @@ function NickelConf.frontLightLevel.get()
         -- FrontLightState config, so don't normalize the value here yet.
         return tonumber(new_intensity)
     else
-        -- In NickelConfSpec, require("device") won't return KoboDevice
-        local powerd = require("device/kobo/powerd")
-        local fallback_fl_level = powerd.fl_intensity or 1
+        local fallback_fl_level = 20
```

Comment:
On the one hand having it bright enough to notice that it came on might be a good idea.  On the other hand, my preferred "fallback" would be off.  (I seem to remember that there was an issue with setting the Kobo frontlight at its lowest setting, perhaps that is why it was on one, as part of the solution.